### DISCUSSION TITLE
Windows Terminal v1.12.1098

### DISCUSSION THREAD

**Discussion by DHowett** [microsoft/terminal#12935] [Announcements]
This release was made available to insiders in the Dev and Beta channels on April 11 and is now generally available.

<details>
<summary>Preinstallation Kit info</summary>

A [preinstallation](https://docs.microsoft.com/en-us/windows/msix/desktop/deploy-preinstalled-apps) kit is available for system integrators and OEMs interested in prepackaging Windows Terminal with a Windows image. More information is available in the [DISM documentation on preinstallation](https://docs.microsoft.com/windows-hardware/manufacture/desktop/preinstall-apps-using-dism). Users who do not intend to preinstall Windows Terminal should continue using the _msixbundle_ distribution.

</details>

<details>
<summary>Why are there so many packages? How do I choose?</summary>
This version of Windows Terminal is distributed in two bundles, one of which works on Windows 10-11 and the other of
which only works on Windows 11. The Windows 11 version is much smaller because we no longer need to work around a
platform issue related to our dependencies.

If you intend on using Terminal as an unpackaged application--that is, extracting the `msix` file--we recommend that
you use the `Win10` bundle. You will need the [Visual C++ runtime redistributable](https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist).

In addition, if you install the _packaged_ version on either Windows 10 or Windows 11, it now depends on [the Visual C++ Universal Runtime Package](https://docs.microsoft.com/en-us/troubleshoot/developer/visualstudio/cpp/libraries/c-runtime-packages-desktop-bridge?msclkid=3e1dd82aac8b11ec8f9dbd86d5979ca8#how-to-install-and-update-desktop-framework-packages).

Despite these distributions having different version numbers, they are built from the same code and there is no
functional difference between them.

If you install the Windows 10 verison on Windows 11, it will probably automatically upgrade itself to the Windows 11
version. It turns out that it is impossible to have two bundles with _the same version number_, so it has to be this
way.
</details>

## Changes

* The refreshed Windows 11 UI from the 1.13 preview builds is now available in 1.12!

### Appearance

* Our Maximize/Restore button is now a fine round boi (#12660)

### Accessibility

* The profile list in the Settings UI now offers tooltips for long profile names (#12448)
* We'll automatically focus the window renamer textbox when it opens (#12798)
* High contrast will no longer result in a ridiculous and bad titlebar color (#12839)
* When you delete a color scheme, we'll move focus back to the color scheme list (#12841)
* When you delete a profile, we will re-focus the delete button automatically (#12558)
* Two instances of huge debug log spam with a screen reader connected have been stamped out (#12698) (#12723)

### Usability

* We've added some text to the color schemes page indicating that it is for editing--not setting--color schemes (#12663)
   * We're working to refine how color schemes are set and edited, so stay tuned for future improvements!
* The retro terminal effect (as well as other shaders) will now work on pre-D3D11 hardware! (#12677)
* Terminal will once again render properly when you move between different-DPI displays (#12713) (#12749)
* Resizing the window while a background color or underline is displayed will no longer smear it across the whole screen (#12637) plus a fix for a huge crash that PR introduced (#12853)
* It took us three releases to get it right, but we've finally solved the issue where we'd punch a hole straight through the Terminal when a dialog appeared (#12840)

### Reliability

* Typing an invalid background image path into the Settings UI will no longer send Terminal to a farm upstate (#11542) (thanks @serd2011!)
* There was an issue on Windows 11 where Terminal would queue up billions of animations while the screen was off; it will now no longer do so (#12820)
* We've fixed crashes in `ProposeCommandline` (#12838), `Monarch::_GetPID` (#12856) and other parts of WT's RPC infrastructure (#12825)
* On Windows 10, the settings UI will no longer sometimes crash on close (we've updated to a new build of WinUI 2 for the fix!) (#12847)

### Miscellaneous

* Windows will no longer reject certain Terminal updates/reinstalls due to "differing package content" (#12779)
* Fragments can once again override the names of generated profiles (#12627)
* An issue from the 1073 series, where you could not upgrade the bundle using DISM, has been resolved (#12819)
   * As a result, our bundle version is now over three thousand!
* @dmezh contributed some wording changes to the text about transparency/opacity (#12592) (#12727) (thanks!)

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/terminal/releases/tag/v1.12.10982.0'>Windows Terminal v1.12.1098</a>.</em>

**Comment by skycommand** [comment]
It appears Windows App Installer mistakenly thinks there is a newer version of 1.12.10983 available. Please note the erroneous 3 at the end of the build number. (Must be 2.)
